index.html

```diff
@@ -57,5 +58,191 @@ <h2 class="supporting-text">I'am a Software Developer</h2>
+                            <p class="group-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <nav>
+                                <ul class="group-ul">
+                                    <li>
+                                        <a class="group-a" href="#">html</a>
+                                    </li>
+                                    <li class="bootstrap">
+                                        <a class="group-a" href="#">bootstrap</a>
+                                    </li>
+                                    <li>
+                                        <a class="group-a" href="#">Ruby</a>
+                                    </li>
+                                </ul>
+                            </nav>
+                        </div>
+                        <button class="action">See Project</button>
+                    </div>
+                    <div class="card-work">
+                        <div class="group">
+                            <h2 class="group-header">Profesional Art Printing Data</h2>
+                            <p class="group-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <nav>
+                                <ul class="group-ul">
+                                    <li>
+                                        <a class="group-a" href="#">html</a>
+                                    </li>
+                                    <li class="bootstrap">
+                                        <a class="group-a" href="#">bootstrap</a>
+                                    </li>
+                                    <li>
+                                        <a class="group-a" href="#">Ruby</a>
+                                    </li>
+                                </ul>
+                            </nav>
+                        </div>
+                        <button class="action">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </article>
+        <aside class="about-me">
+            <h2>About me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button>Get My Resume</button>
+            <div id="lang">
+                <ul id="language">
+                    <li>Language
+                        <a class="skills" href="#">Javascript</a>
+                        <a class="skills" href="#">Ruby</a>
+                        <a class="skills"href="#">Html</a>
+                        <a class="skills" href="#">Css</a>
+                    </li>
```

Comment:
- [ ] I think it would be great if you treat this as a list of "Skills". 📜 You can have an outer container `<ul>` that stores each skill category: Languages, Frameworks, and skills. Each one of them should be a list of their specific items.

I know it's a little tricky, but here's an example skeleton:

```HTML
<ul>
  <li>
    <h3> Languages </h3>
    <ul>
      <li> Language 1 </li>
      <li> Language 2 </li>
      <li> Language 3 </li>
    </ul>
  </li>
  <li>
    <h3> Frameworks </h3>
    <ul>
      <li> Framework 1 </li>
      <li> Framework 2 </li>
      <li> Framework 3 </li>
    </ul>
  </li>
...
</ul>
```

Please consider changing this to improve the semantics of your project! 🛠️ 🤓